src/wallet/wallet_rpc_server_commands_defs.h

```diff
@@ -63,9 +63,14 @@ namespace wallet_rpc
     {
       uint32_t account_index;
       std::set<uint32_t> address_indices;
+      bool all_accounts;
```

Comment:
Need to bump version in this file.

Reply:
fixed.

Reply:
This is now a non-backwards compatible change so should be a major version change.

Reply:
I think I'd rather the new data to be added, and the old one to be still around. This way, it's not incompatible.

Reply:
A new method? I think that's a good idea too.